tests/x11/thunderbird/thunderbird_pop.pm

```diff
@@ -35,6 +36,14 @@ sub run {
     }
 
     mouse_hide(1);
+    # Disable screensaver
```

Comment:
On non-Gnome desktop xterm will be left running.

Screensaver is disabled on multiple places in code as needed,
I think it's better to schedule `disable_screensaver` at beginning. https://openqa.suse.de/tests/5030527#step/disable_screensaver/9
turn_off_gnome_suspend should be needed only on laptops

Reply:
Ok, modified to user disable_screensaver. Please add on yaml scheduled. Tks